content/terraform/v1.14.x/docs/language/backend/azurerm.mdx

```diff
@@ -17,14 +17,14 @@ The `azurerm` backend needs to authenticate to the storage account data plane in
 
 The `azurerm` backend supports 5 methods to authenticate to the storage account data plane:
 
-- [Microsoft Entra ID](#microsoft-entra-id) **(Recommended)**
```

Comment:
I may be misremembering, but I thought this change was the other way around, where Azure AD was changed to Entra AD?